align/compiler/match_graph.py

```diff
@@ -178,11 +179,24 @@ def pin_id(pins):
                     gen_const = constraint.Generator(**const.generator)
                     with set_context(parent_subckt.constraints):
                         new_subckt.constraints.append(gen_const)
+
                 self.ckt_data.append(new_subckt)
             # Add all instances of groupblock to new subckt
             with set_context(new_subckt.elements):
                 for e in new_insts:
                     new_subckt.elements.append(Instance(**e))
+
```

Comment:
@kkunal1408 , below is an example to avoid hard-coding attribute names. Each constraint class, if desired, specified `_instance_attribute` field. See examples in `constraint.py` in this PR.